_posts/2017-07-14_hc-all-devs-meetup.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  "62 Engineers, 1 All-inclusive hotel"
+date: 2017-07-14 14:56:00 +0200
+categories: conference developers openspace
+author_name: Cosima Laube
+author_url : /author/cosimalaube
+author_avatar: cosimalaube
+show_avatar : true
+read_time : 6
+show_related_posts: false
+feature_image: posts/2017-07-14_hc-all-devs-meetup/feature-image.jpg
+---
+
+Early in May this year, all HolidayCheck IT engineers, Scrum masters and line managers from all our locations in Poland, Germany and Switzerland had their regular all-developers meetup. 
+
+## A great unconference location...
+As I was just two months with HolidayCheck back then, I learned one major thing was new about the meetup this time: we all boarded planes with one sunny seaside location... a huge all-inclusive hotel in Turkey.
+
+<img src="{{site.baseurl}}/img/posts/2017-07-14_hc-all-devs-meetup/hcalldevs-venue.jpg" alt="the all developers meetup venue" width="200" style="float: left; margin: 1rem;" />
+
+Of course this location was not chosen just for fun but to feel closer to our customers, the Urlaubers. And it provided decent conditions for conducting two full days of an [Open Space][openspaceWikipedia] unconference mixed with plenty of opportunities to connect with people one just knew from Slack chat or the GitHub commit history.
+
+> Open Space creates a stable, methodic frame, where many people can work on their issues self-organized and self-responsible. The goal is to work on essential, innovative and solution focused subjects in a short time with a big number of people.
+
+## ... and awesome outcomes for real work-life
+
+<img src="{{site.baseurl}}/img/posts/2017-07-14_hc-all-devs-meetup/oneOfTheSessions.jpg" alt="one of the sessions at a restaurant" width="200" style="float: right; margin: 1rem;" />
+
+We had over 40 sessions for both days from discussions to live coding and also creating & incubating new ideas and actions. For example a couple of Communities of Practise (CoP) e.g. on Scala or Testing were started or promoted by these intense days at the beach. Also this tech blog you are currently reading had been reanimated - or to be more precise: rebuilt - as a direct consequence of the gathering.
```

Comment:
Just a little detail but I would suggest to change this:

> Also this tech blog you are currently reading had been reanimated - or to be more precise: rebuilt - as a direct consequence of the gathering

to this:

> The reboot of this very blog was another project that was started during the meetup.

I find the current sentence somewhat long-winded. And `had been` is used to refer to time spans so its use seems a bit off to me here. 

Just my 2cents ;)